packages/flutter/lib/src/foundation/stack_frame.dart

```diff
@@ -0,0 +1,230 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
```

Comment:
2020?

Reply:
All files in this repo are dated to 2014 to keep things consistent